kentikapi/internal/api_resources/sites_test.go

```diff
@@ -0,0 +1,209 @@
+package api_resources_test
+
+import (
+	"testing"
+
+	"github.com/kentik/community_sdk_golang/kentikapi/internal/api_connection"
+	"github.com/kentik/community_sdk_golang/kentikapi/internal/api_resources"
+	"github.com/kentik/community_sdk_golang/kentikapi/internal/utils"
+	"github.com/kentik/community_sdk_golang/kentikapi/models"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestGetSite(t *testing.T) {
+	// arrange
+	getResponsePayload := `
+	{
+		"site": {
+			"id": 42,
+			"site_name": "apitest-site-1",
+			"lat": 54.349276,
+			"lon": 18.659577,
+			"company_id": 3250
+		}
+	}`
+	transport := &api_connection.StubTransport{ResponseBody: getResponsePayload}
+	sitesAPI := api_resources.NewSitesAPI(transport)
+	siteID := models.ID(42)
+
+	// act
+	site, err := sitesAPI.Get(nil, siteID)
+
+	// assert request properly formed
+	assert := assert.New(t)
+	require := require.New(t)
+
+	require.NoError(err)
+	assert.Zero(transport.RequestBody)
```

Comment:
Please check the request path passed to transport.

In other tests too

Reply:
Will add this once transport.RequestPath added in Plans is on master